packages/legacy/core/App/constants.ts

```diff
@@ -1,5 +1,14 @@
+import { Dimensions, Platform } from 'react-native'
+import StaticSafeAreaInsets from 'react-native-static-safe-area-insets'
+
 import { PINValidationRules } from './types/security'
 
+export const SCREEN_WIDTH = Dimensions.get('window').width
+export const SCREEN_HEIGHT = Platform.select<number>({
+  android: Dimensions.get('screen').height - StaticSafeAreaInsets.safeAreaInsetsBottom,
+  ios: Dimensions.get('window').height,
+}) as number
+
```

Comment:
How does this behave on tablets when they switch between portrait and landscape on the camera screen?

Reply:
good question, BC Wallet is locked in portrait mode so it shouldn't be an issue

Reply:
I don't think it's locked in portrait mode on tablets, only on phones

Reply:
I tested it on the pixel tablet with auto rotate turned on and the BC Wallet app was locked in portrait mode on that device

Reply:
Hmm. Maybe it's just unlocked on iPads. We should ask @jleach

Reply:
@bryce-mcmath it works fine on Ipads in either orientation, the only issue is that in landscape mode moving the camera is inverted, so left is down, right is up, etc. Trying to see if I can fix that issue now

Reply:
Seems like this is a known issue with the library:
- https://github.com/mrousavy/react-native-vision-camera/issues/1455
- https://github.com/mrousavy/react-native-vision-camera/issues/1891

Reply:
I could downgrade to version 2 or switch to another library but that may introduce it's own dependency issues and IMHO that seems like a lot of work to address what would be a minor inconvenience to only a handful of users, especially when it looks like this issue is going to be fixed in the library eventually. What are your thoughts @bryce-mcmath @jleach ?

Reply:
I would tend to agree with that but what happened with testing older Android versions that we still support? I remember you saying something like API 24 wasn't scanning

Reply:
Actually I just tested it again on my mac environment and it worked fine on API 24. I think maybe I just didn't try moving the camera around last time

Reply:
Well then I'm ok to move forward with this. Might want to wait until Clecio gets back and takes a look at it but I think you've done a great job